app/src/main/java/com/monkeyteam/chimpagne/ui/components/eventview/OrganiserView.kt

```diff
@@ -62,17 +61,8 @@ fun OrganiserView(
       horizontalArrangement = Arrangement.SpaceBetween,
       modifier = Modifier.fillMaxWidth().padding(horizontal = 16.dp)) {
         Row {
-          profilePictureUriState.value?.let { uri ->
-            ProfileIcon(
-                uri = uri,
-                onClick = {
-                  Toast.makeText(
-                          context,
-                          "This function will be implemented in a future version",
-                          Toast.LENGTH_SHORT)
-                      .show()
-                })
-          } ?: CircularProgressIndicator()
+          profilePictureUriState.value?.let { uri -> ProfileIcon(uri = uri, {}) }
```

Comment:
if the ProfileIcon isn't clickable, set enable to false

Reply:
you are right, I missed that. Thank you